### ISSUE TITLE
Theme uses some Permalink instead of RelPermalink

### ISSUE THREAD

**Issue by Dekker500** [hugo-toha/toha#579]
Don't know if it was done for a particular reason, but some pages incorporate manually-crafted links instead of using "RelPermalink".

For example, when setting the background image, toha/layouts/partials/sections/home.html has numerous 
`url('{{ strings.TrimSuffix "/" site.BaseURL }}{{ $src.RelPermalink }}')`
statements

These should ideally be replaced by
`url('{{ $src.RelPermalink }}')`

No reason to have the full URL.

**Comment by hossainemruz** [comment]
>These should ideally be replaced by
url('{{ $src.RelPermalink }}')

It does not worked. Tried it before.

**Comment by Dekker500** [comment]
Interresting. The RelPermalink works perfectly for me in a production site (http://www.dedicatedtovo.com/), and I have used the same sort of link before. In fact, I don't know why it would be any different than any of the other img links in the theme. 

I discovered the issue when the staging site (not the official site) could not load the background image because the links were hard-coded with the site URL. 

If the solution is not usable by everyone, so be it.
